gems/sorbet/test/snapshot/BUILD

```diff
@@ -23,10 +23,10 @@ sh_binary(
         "sorbet-typed.rev",
         "//main:sorbet",
         "@gems//gems",
-        "@ruby_2_6//:bundle",
-        "@ruby_2_6//:ruby",
-        "@ruby_2_7//:bundle",
-        "@ruby_2_7//:ruby",
+        "@sorbet_ruby_2_6//:bundle",
+        "@sorbet_ruby_2_6//:ruby",
+        "@sorbet_ruby_2_7//:bundle",
+        "@sorbet_ruby_2_7//:ruby",
```

Comment:
Is it worth trying to run these tests with an unpatched version of ruby, instead of sorbet_ruby?

Reply:
Possibly? I don't think that the patching changes the build time dramatically, and our goal has been that there aren't any changes to non-compiled code-paths with our patches.